_posts/Ibbi2010/2017-05-22-Ibbi2010-turtlehack.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+author: Ibbi2010
+title: "Ibbi2010's turtle exercise"
+---
+
+My initial experience with the trinket was almost no experience. It took me a little while to explore what a turtle could do. Once that happened, I wanted to try as many possibilities as I could. But keeping in view my limited knowledge/vocabulary about python, I wanted to polish what I had already learnt. which was mainly creating shapes by using left, right, angles with varying degrees and “forward” codes. 
```

Comment:
Good goals, and a great start to your reflection!  Thinking critically about what you want to get out of an assignment should help you cut through any obstacles you encounter.